helpers/cuenta_bancaria_banco.go

```diff
@@ -0,0 +1,106 @@
+package helpers
+
+import (
+	"encoding/json"
+	"strconv"
+
+	"github.com/udistrital/tesoreria_mid/models"
+
+	giros_crud "github.com/udistrital/giros_crud/models"
+	terceros_crud "github.com/udistrital/terceros_crud/models"
+	errorctrl "github.com/udistrital/utils_oas/errorctrl"
+)
+
+// Obtiene una cuenta bancaria por Id
+func ObtenerCuentaBancariaBancoPorId(id int) (res *models.CuentaBancariaBanco, err map[string]interface{}) {
+	funcion := "ObtenerCuentaBancariaBancoPorId"
+	defer errorctrl.ErrorControlFunction(funcion, "500")
+	cuentaBancaria := models.CuentaBancariaBanco{}
+	// Consulta en giros_crud de cuentas bancarias
+	if cuenta, err := ObtenerCuentaBancariaPorId(id, &cuentaBancaria); err == nil {
+		// Consulta en terceros_crud
+		_, err := ObtenerInfoComplementariaPorId(cuenta.SucursalId, &cuentaBancaria)
+		if err != nil {
+			return nil, err
+		}
+		return &cuentaBancaria, nil
+	} else {
+		return nil, err
+	}
+}
+
+// Obtiene múltiples cuentas bancarias
+func ObtenerCuentasBancariasBancos(limit int, offset int) (cuentasBancarias []models.CuentaBancariaBanco, err map[string]interface{}) {
+	funcion := "ObtenerCuentasBancariasBancos"
+	defer errorctrl.ErrorControlFunction(funcion, "500")
+	var cuentas []giros_crud.CuentaBancaria
+	if err := GetAll(&cuentas, "giros_crud", "cuenta_bancaria", 2, nil, nil, nil, nil, limit, offset); err == nil {
+		cuentasBancarias := make([]models.CuentaBancariaBanco, len(cuentas))
+		for i, cuenta := range cuentas {
+			cuentaBancaria := models.CuentaBancariaBanco{}
+			SetCuentaBancariaBancoPorCuentaBancaria(&cuenta, &cuentaBancaria)
+			if _, err := ObtenerInfoComplementariaPorId(cuenta.SucursalId, &cuentaBancaria); err != nil {
+				return nil, err
+			}
+			cuentasBancarias[i] = cuentaBancaria
+		}
+		return cuentasBancarias, nil
+	} else {
+		return nil, err
+	}
+}
+
+// Obtención
+
```

Comment:
```suggestion
// ObtenerCuentaBancariaPorId ...
```

Tener cuidado que el comentario anterior, la descripción de la función; no quede con el espacio que está actualmente en medio, eliminar la L54

Reply:
Se ajusta en commit https://github.com/udistrital/tesoreria_mid/commit/0411cccd93d5f8cf92ca51b6b302fae4ddb57e5e